build-system/global-configs/canary-config.json

```diff
@@ -59,6 +59,5 @@
   "no-sync-xhr-in-ads": 1,
   "pump-early-frame": 1,
   "sandbox-ads": 1,
-  "user-error-reporting": 1,
-  "version-locking": 1
```

Comment:
I think you have to make these in a separate PR.

Reply:
Will do.